roles/prometheus/templates/prometheus.yaml.j2

```diff
@@ -0,0 +1,14 @@
+    global:
+      scrape_interval: 30s
+      evaluation_interval: 10s
+    scrape_configs:
+      - job_name: "http_sd_hosts"
+        honor_timestamps: true
+        scrape_interval: 30s
+        scrape_timeout: 30s
+        scheme: http
+        follow_redirects: true
+        http_sd_configs:
+        - follow_redirects: true
+          refresh_interval: 1m
+          url: http://localhost:{{ web_container_port }}/api/prometheus/targets
```

Comment:
I don't like this localhost that much.
I think we should have this as a default value in `defaults.yaml` (`webhost: http://localhost`) or so, and use it here.
Otherwise, if prometheus is running in other host, it wouldn't work